CHANGELOG.md

```diff
@@ -1,17 +1,27 @@
 # Changelog
-## v2.3.0
+## v2.3.0 - 16 December 2024
```

Comment:
I've missed this one previously; like previous entries, let's use short form
```suggestion
## v2.3.0 - 16 Dec 2024
```

Reply:
Done 👍